integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -578,6 +582,58 @@ void testAdminServiceAnnotationsLabels() {
         "Missing expected label on admin service");
   }
 
+  /** The default replicas at domain resource level is 1. Patching domain1 with new replicas 3.
+   * Verify managed server pods count at cluster resource level are not affected by domain resource
+   * level patching.
+   * Replicas count at cluster resource level is defined as 2.
+   * Scale up the cluster by patching replicas count to 5 at cluster resource level.
+   * Scale down the cluster by patching replicas count to 1 at cluster resource level.
+   * Verify cluster resource level scaling is successful even though replicas count at domain resource
```

Comment:
The first line of description says "replicas at domain resource level is 1"

Reply:
Yes, that is default seting

Reply:
Yes, that is the default setting.